packages/gatsby-theme-api-docs/src/design-system.js

```diff
@@ -35,5 +41,7 @@ export const typography = {
     th: designSystem.pxToRem('21px'),
```

Comment:
This looks like a magic number. Please use a "normal" value, like `20` or `24`

Reply:
I got it from the design.

Reply:
I know, but we as developers should not just take everything for granted and bring such things to the attention of the designer that gave the number to you. Same goes for colors, etc.
You need to keep an eye out for this things and when you see such "magic numbers" it usually means that something does not feel right.

Please check back with Sven about this, thanks.